doc/api/test.md

```diff
@@ -1645,9 +1643,12 @@ changes:
   `fn` does not have a name.
 * `options` {Object} Configuration options for the subtest. The following
   properties are supported:
-  * `concurrency` {number} The number of tests that can be run at the same time.
+  * `concurrency` {number|boolean} If a number is provided,
+    then that many files would run in parallel.
+    If `true`, it would run `os.availableParallelism() - 1` files in parallel.
+    If `false`, it would only run one test at a time.
     If unspecified, subtests inherit this value from their parent.
-    **Default:** `1`.
+    **Default:** `false`.
```

Comment:
```suggestion
    **Default:** `null`.
```

Reply:
Isn’t it because the parent has a concurrency of `1`/`false`?

Reply:
If the parent has concurrency turned off by default via `1`/`false` then the subtest should also have that value, right?

Reply:
The fact that the default value for the parent is `false` is irrelevant imo, we are not documenting what the actual value will be if only defaults are being used, we are describing the API behavior. Wouldn't you agree that it would be confusing for someone that's using something that's not `1` in the parent test? If we say in the docs that `null` means "inherit the value from the parent", then the default value is `null`, not `false` nor `1`. Does that make sense?

Reply:
Sure, would this logic also apply to the other places where values are inherited from the parent? like `options.timeout` in the various places?

Reply:
Yes I think that would apply the same. I suspect the current text is a result of a copy pasting and the mistake slip through. We can either keep this PR focused on `concurrency` and open a PR in parallel to fix the other options, or do that in this PR and change the commit message.

Reply:
Yes, I'll make the changes you've suggested here and keep this PR focused on `concurrency`. I'll create a follow up PR to fix the other options.